README.md

```diff
@@ -1,5 +1,16 @@
-# webauthn
-[![Docs](https://pkg.go.dev/badge/github.com/pomerium/webauthn?status.svg)](https://pkg.go.dev/github.com/pomerium/webauthn?tab=doc)
-[![Coverage Status](https://coveralls.io/repos/github/pomerium/webauthn/badge.svg)](https://coveralls.io/github/pomerium/webauthn)
+# Webauthn
 
-Go WebAuthn Library
+[![Go Report Card](https://goreportcard.com/badge/github.com/pomerium/webauthn)](https://goreportcard.com/report/github.com/pomerium/webauthn) [![GoDoc](https://pkg.go.dev/badge/github.com/pomerium/webauthn?status.svg)](https://pkg.go.dev/github.com/pomerium/webauthn?tab=doc) [![Coverage Status](https://coveralls.io/repos/github/pomerium/webauthn/badge.svg)](https://coveralls.io/github/pomerium/webauthn) [![pomerium chat](https://img.shields.io/badge/chat-on%20slack-blue.svg?style=flat&logo=slack)](http://slack.pomerium.io) [![GitHub Actions](https://img.shields.io/github/workflow/status/pomerium/webauthn/Release?style=flat)](https://github.com/pomerium/webauthn/actions?query=workflow%3ATest) [![LICENSE](https://img.shields.io/github/license/pomerium/webauthn.svg)](https://github.com/pomerium/webauthn/blob/master/LICENSE)
```

Comment:
Not sure why coveralls is showing unknown (here and on their site).

Reply:
Ahh, it's because these were only being run on "main" and we are still using "master" for our primary development branch.